extension/package.json

```diff
@@ -42,6 +42,11 @@
         "command": "xcomponent.preview.composition",
         "title": "XComponent: Preview Composition",
         "group": "navigation"
+      },
+      {
+        "command": "xcomponent.preview.spy",
```

Comment:
it's not a preview... it should be on its own namespace... like `xcomponent.launch.spy`